PrivateTalkApp/Common/Services/WorldTimeService.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WorldTimeService.swift
+//  PrivateTalkApp
+//
+//  Created by 都甲裕希 on 2024/08/27.
+//
+
+import Foundation
+
+final class WorldTimeService {
+    
+    private let worldTimeRepository = WorldTimeRepository()
+    
+    func fetchWorldTimeApi() async throws -> WorldTimeResponse {
+        let worldTimeResponse = try await worldTimeRepository.fetchCurrentTime()
+        // レスポンスのdatetimeを正しい文字列に変換
+        let correctDateString = trimDateString(dateString: worldTimeResponse.datetime)
+        // 正しい文字列でレスポンスを再生成
+        let newWorldTimeResponse = worldTimeResponse.updatedDatetimeProperty(newDatetime: correctDateString)
+        
+        return newWorldTimeResponse
```

Comment:
ここの実装がちょっとごちゃついてる感じした。
`fetchWorldTime()`が何をするのかはよくわからんが、時間のStringを取得する関数っぽいから別に`WorldTimeResponse`ではなく`String`を返せば良さそう。

```suggestion
        let response = try await worldTimeRepository.fetchCurrentTime()
        // レスポンスのdatetimeを正しい文字列に変換
        return trimDateString(dateString: response.datetime)
```

Reply:
そうすれば、`updatedDatetimeProperty(newDatetime: )`は実装しなくて良さそう。

Reply:
Fixed.